### ISSUE TITLE
JS errors don't show the correct location when clicked in the console

### ISSUE THREAD

**Issue by CDK2020** [flarum/core#1538]
## Bug Report

**Current Behavior**
A clear and concise description of the behavior.

**Steps to Reproduce**
1. Add error to js dist file
2. Look for error on js console
3. Click wrong location

**Expected Behavior**
It should show where the error in JS is

**Screenshots**
![image](https://user-images.githubusercontent.com/13856015/44006155-3fc5f142-9e34-11e8-8fac-0862cfe0c73a.png)

![image](https://user-images.githubusercontent.com/13856015/44006160-4d5e153c-9e34-11e8-929c-fb1a0c0e043e.png)

**Environment**
- Flarum version: dev-database-changes
- Browser: Chrome 68

**Possible Solution**
Could be related to Webpack maps.

**Comment by datitisev** [comment]
I can reproduce this. Sometimes it shows core components and then ends in `admin.js:1` or `forum.js:1` instead of going further as well.

**Comment by CDK2020** [comment]
I've identified the problem and I am working on a fix, expect a PR soon.

**Comment by tobscure** [comment]
@CDK2020 discovered a lingering issue with conflicting sourcemaps between extensions - basically the map for each extension's index.js file was overwriting the previous.

I have fixed this in https://github.com/flarum/flarum-webpack-config/releases/tag/v0.1.0-beta.9. Core has been updated; would appreciate PRs for extensions (just need to run `npm update` and commit `package.json` and `package-lock.json`).
